tests/unit_node/util_test.ts

```diff
@@ -315,3 +315,10 @@ Deno.test({
     );
   },
 });
+
+Deno.test({
+  name: "[util] parseArgs() with no args works",
+  fn() {
+    util.parseArgs();
```

Comment:
This looks causing a type error. I think we need to pass an empty object
```suggestion
    util.parseArgs({});
```

Reply:
yep, that was the problem. I've fixed it in my fork. I've seen the CI test that fails is run with `cargo test`. However, that takes a lot of time to run in my machine. What's the most appropriate way I should be running tests? I've used previously the `./target/debug/deno test --config tests/config/deno.json -A tests/unit_node/util_test.ts` command, which run only the tests of my PR. However, that didn't check for TS syntax error it seems.